src/main/java/LocalStorageTranslator.java

```
@@ -98,30 +98,30 @@ public void saveSettings(Map<String, String> _settings, ArrayList<String> _param
      * @param _input  Array List of String containing the data from API output to be stored.
      */
     public void store(ArrayList<String> _params, ArrayList<String> _input) {
-        String fileName = _params.get(0);
-        Path filePath = Paths.get(fileName);
+        String fileName = getFileName(_params);
+        Path filePath = getPath(fileName);
 
         if (Files.exists(filePath)) {
             System.out.println(fileName + " exists and the data in it will be overwritten.");
         }
 
         try {
             File file = new File(fileName);
-            PrintWriter pw = new PrintWriter(file);
+            PrintWriter writer = new PrintWriter(file);
 
             // loop through _input and write to file
             for (int i = 0; i < _input.size(); i++) {
 
-                // ensure no trailing whitespace
+                // ensure no trailing whitespace or it WILL cause an error when reading back
                 if (i == _input.size() - 1) {
-                    pw.print(_input.get(i));
+                    writer.print(_input.get(i));
                 } else {
-                    pw.println(_input.get(i));
+                    writer.println(_input.get(i));
                 }
             }
-//            pw.println(_input.toString());
-            pw.flush();
-            pw.close();
+
+            writer.flush();
+            writer.close();
         } catch (IOException ex) {
```

If storing the file fails, lets throw an exception.